src/coreclr/tools/superpmi/superpmi/icorjitinfo.cpp

```diff
@@ -108,8 +113,11 @@ void MyICJI::reportInliningDecision(CORINFO_METHOD_HANDLE inlinerHnd,
                                     CorInfoInline         inlineResult,
                                     const char*           reason)
 {
-    jitInstance->mc->cr->AddCall("reportInliningDecision");
-    jitInstance->mc->cr->recReportInliningDecision(inlinerHnd, inlineeHnd, inlineResult, reason);
+    if (inlineResult == INLINE_PASS || inlineResult == INLINE_FAIL|| inlineResult == INLINE_NEVER)
+    {
```

Comment:
Seems like we don't need this condition

Reply:
Yep. I've removed it and things seem to work.